src/OceanSeaIceModels/time_step_ocean_sea_ice_model.jl

```diff
@@ -39,9 +39,9 @@ function time_step!(coupled_model::OceanSeaIceModel, Δt; callbacks=[], compute_
     #  - Adjust ocean heat flux if the ice completely melts?
     ocean.Δt = Δt
     time_step!(ocean)
```

Comment:
```suggestion
    time_step!(ocean, Δt)
```
use this to ensure that the ocean simulation takes a time step of Δt. Otherwise it is not guaranteed which could be the source of issues for you.